### DISCUSSION TITLE
non appimage binary for Linux as official releases

### DISCUSSION THREAD

**Discussion by CrypticEntity** [ppy/osu#14380] [Ideas] [answered]
I don't know how easy this would be to implement. Having an rpm/deb build of Lazer would go a long way to getting official package manager support. (I also just dislike appimages generally)

**Comment by bdach** [comment, marked as answer]
The appimage is deliberately the only officially supported distribution method of the game on linux. The linux ecosystem is too fragmented to maintain however many packages in however many package registries various distributions use.
